src/pages/cash-count/forms/GenerateTransferForm.js

```diff
@@ -0,0 +1,122 @@
+import { Grid, Grow } from '@mui/material'
+import React, { useContext } from 'react'
+import FormShell from 'src/components/Shared/FormShell'
+import ResourceComboBox from 'src/components/Shared/ResourceComboBox'
+import { ResourceLookup } from 'src/components/Shared/ResourceLookup'
+import { useForm } from 'src/hooks/form'
+import { RequestsContext } from 'src/providers/RequestsContext'
+import { CashBankRepository } from 'src/repositories/CashBankRepository'
+import { CashCountRepository } from 'src/repositories/CashCountRepository'
+import { SystemRepository } from 'src/repositories/SystemRepository'
+import { ResourceIds } from 'src/resources/ResourceIds'
+import * as yup from 'yup'
+import toast from 'react-hot-toast'
+import { VertLayout } from 'src/components/Shared/Layouts/VertLayout'
+import { Fixed } from 'src/components/Shared/Layouts/Fixed'
+
+export default function GenerateTransferForm({ cashCountId, fromPlantId, labels, maxAccess }) {
+  const { postRequest } = useContext(RequestsContext)
+
+  const { formik } = useForm({
+    maxAccess,
+    initialValues: {
+      cashCountId: cashCountId,
+      fromPlantId: fromPlantId,
+      toPlantId: '',
+      toAccountId: ''
+    },
+    enableReinitialize: false,
+    validateOnChange: true,
+    validationSchema: yup.object({
+      toPlantId: yup.string().required(' '),
+      toAccountId: yup.string().required(' ')
+    }),
+    onSubmit: async obj => {
+      console.log(obj)
+
+      const data = {
+        cashCountId,
+        fromPlantId,
+        toPlantId: obj.toPlantId,
+        toAccountId: obj.toAccountId
+      }
+      try {
+        await postRequest({
+          extension: CashCountRepository.CashCountTransaction.generate,
+          record: JSON.stringify(data)
+        })
+        if (!obj.recordId) {
```

Comment:
here it's not a new record to verify if it's added or edited
we will have 1 message and it's "Cash Transfer Generated Successfully"